components/scanners/osv-scanner/component.yaml

```diff
@@ -0,0 +1,22 @@
+name: osv-scanner
+description: "Scans projects with google's osv-scanner, parses sarif findings into OCSF format"
+type: scanner
+steps:
+  - name: scanner
+    image: components/scanners/osv-scanner/scanner
+    executable: /entrypoint.sh
```

Comment:
this will not work, the executable should be `/bin/bash` and then the args should be
```
- -c
- /entrpoint.sh
- scan source -r --format=sarif --call-analysis=true {{ sourceCodeWorkspace }}
```

Reply:
are you saying, this won't work on smithy-pro or just the oss?
for reference it works on the oss